crates/ibc/src/core/ics04_channel/commitment.rs

```diff
@@ -69,3 +69,52 @@ impl From<Vec<u8>> for AcknowledgementCommitment {
         Self(bytes)
     }
 }
+
+pub use compute::{compute_ack_commitment, compute_packet_commitment};
```

Comment:
why not just remove the `compute` module altogether?

Reply:
To keep it modular (achieves a different goal than the code above), better-organized import calls and reusable if needed in other applications later.
It's okay if we keep this, I'll also put `pub(crate)` before `mod compute` ?

Reply:
I don't like that there are now 2 paths to the same items, and both are used internally (e.g [from commitment](https://github.com/cosmos/ibc-rs/blob/450822a47c7040645ed342c4c60d81911f0c4b1a/crates/ibc/src/core/ics04_channel/handler/send_packet.rs#L3), and [from compute](https://github.com/cosmos/ibc-rs/blob/450822a47c7040645ed342c4c60d81911f0c4b1a/crates/ibc/src/core/ics04_channel/handler/recv_packet.rs#L3)).

> achieves a different goal than the code above

I think they do achieve the same goal: the `commitment` module holds structs and free functions related to packet commitments. I think this is also why you think it makes sense to expose it as such to the users ;).

I guess the TLDR of my opinion is that separating free functions from struct definitions is not sufficient to justify a new submodule. However this is subject to endless debate and is more a matter of preference anyways, so I'll let you make the final decision as to whether we keep the module or not!

Reply:
That's fine :smile:
[587fa8b](https://github.com/cosmos/ibc-rs/pull/492/commits/587fa8bab4b78dd48f77a3af70f1954943164e44)